src/main/java/google/calendar/EventBuilder.java

```diff
@@ -0,0 +1,47 @@
+package google.calendar;
+
+import java.util.Calendar;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventDateTime;
+
+/**
+ * @author Matan
+ * Class for constructing a google-event type. Contains the options of writing a begin & end data
+ * Set the description and the title.
+ * Further enhancement can be done by using the api from:
+ * https://developers.google.com/resources/api-libraries/documentation/calendar/v3/java/latest/com/google/api/services/calendar/model/Event.html#getSummary--
+ * 
+ */
+public class EventBuilder {
+	private String title;
+
+	private String description;
+	private EventDateTime start;
+	private EventDateTime end;
+	
+	public EventBuilder setEndDate(Calendar x) {
+		x.add(Calendar.DATE, 1);
```

Comment:
I just want to make sure I really got it. So, does Calendar serve as sort of a type conversion?

Reply:
It just seems a bit conspicuous to me since it is then converted to a DateTime. Why not use DateTime directly?

Reply:
That's a good question. the documentation was. a bit lacking but I will generalize it accordingly.

Reply:
The class DateTime is a Google class.
Instead, we use Calendar which is a class in java.utils

Reply:
@ShaiZeevi java's date c'tor is deprecated and Calendar is to be used instead.